src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/RecommendationControllerTest.kt

```diff
@@ -42,22 +42,32 @@ class RecommendationControllerTest() : IntegrationTestBase() {
     userAccessAllowed(crn)
     mappaDetailsResponse(crn, category = 1, level = 1)
     allOffenderDetailsResponse(crn)
+
```

Comment:
Does this need 2 versions of this test? One with feature on and one off? For the one that is on, we should probably assert the new fields are being set correctly.